cfy_manager/components/validations.py

```diff
@@ -174,6 +175,31 @@ def _validate_inputs():
             )
 
 
+def _validate_host(host, input_name):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        sock.connect((host, INTERNAL_REST_PORT))
+    except socket.error as e:
+        _errors.append(
+            'Could not connect to the {input_name} ({host}) via port {port}\n'
+            'Error: {error}\n'
+            'Check that the IP/hostname is written correctly, is '
+            'reachable and that port {port} is open'.format(
+                input_name=input_name,
+                host=host,
+                error=e,
+                port=INTERNAL_REST_PORT
+            )
+        )
+
+
+def _validate_hosts():
+    logger.info('Validating private and public IPs...')
+
+    _validate_host(config[MANAGER][PRIVATE_IP], 'private IP')
+    _validate_host(config[MANAGER][PUBLIC_IP], 'public IP')
```

Comment:
this must not raise a fatal error because the public IP isn't necessarily routable from the manager